MAUI/Popup/layout-customizations.md

```diff
@@ -0,0 +1,451 @@
+---
+layout: post
+title: Layout Customizations in .NET MAUI Popup control | Syncfusion
+description: Learn here all about Layout Customizations support in Syncfusion .NET MAUI Popup (SfPopup) control, its elements and more.
+platform: MAUI
+control: SfPopup
+documentation: ug
+---
+
+# Layout Customizations in MAUI Popup (SfPopup)
+
+## Popup appearance mode for footer
+
+The `SfPopup` supports two types of `SfPopup.AppearanceMode`. By default, the `PopupButtonAppearanceMode.OneButton` is set. You can change the appearance by using the `SfPopup.AppearanceMode` property.
+
+The two different appearance modes in the SfPopup are as follows:
+<table>
+<tr>
+<th>Modes</th>
+<th>Description</th></tr>
+<tr>
+<td>`OneButton`</td>
+<td>Shows the `SfPopup` with one button (accept button) in the footer view. This is the default value.</td></tr>
+<tr>
+<td>`TwoButton`</td>
+<td>Shows the `SfPopup` with two buttons (accept & decline buttons) in the footer view.</td></tr>
+</table>
+
+In the following code example, set the `SfPopup.AppearanceMode` property as `OneButton` which displays only the Accept button in the footer view.

```

Comment:
In the following code example, set the `SfPopup.AppearanceMode` property as `OneButton,` which displays only the Accept button in the footer view.